ts/state/ducks/networkData.ts

```diff
@@ -107,7 +107,7 @@ const refreshInfoFromSeshServer = createAsyncThunk(
       payloadCreator.dispatch(setInfoFakeRefreshing(true));
       await sleepFor(0.5 * DURATION.SECONDS);
       payloadCreator.dispatch(setInfoFakeRefreshing(false));
-      payloadCreator.dispatch(setLastRefreshedTimestamp(Date.now()));
+      payloadCreator.dispatch(setLastRefreshedTimestamp(NetworkTime.now()));
```

Comment:
I am actually not too sure we want to update those to be NetworkTime in this file.
The reason is that I assume this field is shown as an "xxx ago" somewhere in the UI, and if the user is not in sync with the network it will always be appear to be late.
Like you just hit refresh and it displays "updated 5 minutes ago" Would be a bit weird.

Reply:
bump for this comment

Reply:
Thought i reverted this, have reverted now :pray: